core-web/libs/edit-content/src/lib/components/dot-edit-content-form/dot-edit-content-form.component.scss

```diff
@@ -5,6 +5,12 @@
     grid-template-columns: 1fr 16rem;
     gap: $spacing-4;
     padding: $spacing-4;
+    padding-bottom: 0;
+    height: 100%;
```

Comment:
Why?

Reply:
The height to force the scroll and the padding-bottom because it was cutting of the form while scrolling

Reply:
Hacky... we need to do something different because the sidebar will need scroll as well.